.travis.yml

```diff
@@ -84,6 +84,8 @@ notifications:
     on_success: always
     on_failure: always
     on_start: always
+  slack:
+    secure: "DHHSNmiCf71SLa/FFSqx9oOnJjJt2GHYk7NsFIBb9ZY10RvQtIPfaoNxkPjqu9HLyZWJSFtg/uNKKplEHc6W80NoXyqoTvwOxTPjMaViXaCNqsmzjjR/JaCWT/oWGXyAw0VX3S8cwuIexlKQGgZwJpIzoVOZqUrDrHI/O17kZoM="
```

Comment:
Can you add a comment that you created the weburl, so we know who to contact in case it does not work anymore.

Reply:
@robkooper Done. For the record it appears that all Slack users have access to see and change the token, but encrypting it does require one extra step -- I used `travis encrypt "pecanproject:THE_TOKEN_HERE" --add notifications.slack -r PecanProject/pecan` -- and I'm happy to be the point person for doing it again next time.